pkg/templates/nodelocaldns/nodelocaldns.go

```diff
@@ -98,6 +98,8 @@ func Deploy(s *state.State) error {
 		return errors.New("kubernetes client not initialized")
 	}
 
+	s.Logger.Infoln("Ensure node local DNS…")
```

Comment:
I think the feature is called NodeLocalDNSCache. I'm not sure how exactly it is written, but I think there is cache in the name.